clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -124,55 +84,47 @@ object IoUtil extends IoUtil {
     */
   class GsUtil {
```

Comment:
If we're doing cleanup, I'd rather get rid of this nested class entirely. It was only separate so we could inject mock standard output for testing our regexes.